package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "vega-editor-backend",
+  "version": "1.0.0",
```

Comment:
Let's start with pre 1.0 for now.

Reply:
@domoritz I think we should follow the [semantic versioning guidelines](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase). Shall I rewrite it to 0.1.0?

Reply:
Yes, we do follow it everywhere. 0.1 sounds good.